sys/auto_init/saul/auto_init_sps30.c

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (C) 2020 HAW Hamburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     sys_auto_init_saul
+ * @brief       Auto initialization of Sensirion SPS30 device driver
+ * @author      Michel Rottleuthner <michel.rottleuthner@haw-hamburg.de>
+ * @file
+ */
+
+#ifdef MODULE_SPS30
+
+#include "assert.h"
+#include "log.h"
+#include "saul_reg.h"
+#include "sps30.h"
+#include "sps30_params.h"
+
+/**
+ * @brief   Define the number of configured sensors
+ */
+#define SPS30_NUM               ARRAY_SIZE(sps30_params)
```

Comment:
duplicate, can't this be defined in `sps30_params.h`?

Reply:
done